.github/workflows/release.yml

```diff
@@ -0,0 +1,69 @@
+name: Build and upload to PyPI
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
+  cancel-in-progress: true
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    outputs:
+      SDIST_NAME: ${{ steps.sdist.outputs.SDIST_NAME }}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # We need the full history to generate the proper version number
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.11'
+
+      - name: Install dependencies
+        run: python -m pip install build twine
+
+      - name: Build sdist
+        id: sdist
+        run: |
+          python -m build --sdist
+          # Get the name of the build sdist file for later use
+          echo "SDIST_NAME=$(ls -1 dist)" >> $GITHUB_OUTPUT
+
+      - name: Check README rendering for PyPI
+        run: twine check dist/*
+
+      - name: Upload sdist result
+        uses: actions/upload-artifact@v3
+        with:
+          name: sdist
+          path: dist/*.tar.gz
+          if-no-files-found: error
+
+  pypi-publish:
+    name: Upload release to PyPI
+    runs-on: ubuntu-latest
+    needs: build_sdist
+    environment:
+      name: pypi
+      url: https://pypi.org/p/swxsoc
```

Comment:
Will this populate after the build and release?

Reply:
yes